core/models/fields/RFIDField.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class RFIDField(models.CharField):
+    """
+    Very simple model field specifically designed to store RFID's.
+
+    Essentially just a thin wrapper around CharField that forces the RFID to be unique and ensures that it has at most
+    10 characters.
+    """
+
+    description = "Specific type of charfield for storing 10 digit RFID's"
+
+    def __init__(self, *args, **kwargs):
```

Comment:
`*args` and `**kwargs` can be scary for new programmers. Are you sure you need to pass a variable number of arguments to the function?

Reply:
This is the way Django recommends doing it.

https://docs.djangoproject.com/en/2.0/ref/models/fields/

It allows users further down the road to use the options on the underlying Char field, such as default value, validators, etc 

https://docs.djangoproject.com/en/2.0/ref/models/fields/